pkg/bidstrategy/semantic/networking.go

```diff
@@ -17,6 +17,7 @@ func NewNetworkingStrategy(accept bool) *NetworkingStrategy {
 }
 
 const (
+	// TODO(forrest) [correctness] this link is no longer valid
```

Comment:
Is it worth updating this, or are you expecting it to change.  
It might be best to remove it rather than leave a broken link in?

Reply:
I'd prefer we avoid embedding documentation links in our error messages as they are bound to become out of data in the future. I'll remove it.

Reply:
a04ebb89807cf32a34d7a5d2a0af76bd8d74b0fe

Reply:
> I'd prefer we avoid embedding documentation links in our error messages as they are bound to become out of data in the future. I'll remove it.

Yeah, it feels like that's a different service that maps error codes to documentation URLS.  +1 on removal.